src/styles/blocks/testimonial.scss

```diff
@@ -0,0 +1,50 @@
+.testimonial {
+
+  &__photo-container {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    column-gap: 24px;
+    margin-bottom: 28px;
+
+    @include desktop {
+      margin-bottom: 32px;
+    }
+  }
+
+  &__photo {
+    grid-column: 2 / 4;
+    width: 100%;
```

Comment:
let's not change photo size across different device types, just change the testimonials layout, because on tablets photos look too big 

<img width="505" alt="Screenshot 2021-09-06 at 11 31 22" src="https://user-images.githubusercontent.com/57598825/132186725-fccffee5-a8b0-4939-95ed-2e3aae38cb5c.png">

Reply:
added max-width 500px on photo in testimonial